modules/ocf_irc/manifests/nodejs/apt.pp

```diff
@@ -0,0 +1,18 @@
+class ocf_irc::nodejs::apt {
+  package {
+    'apt-transport-https':;
+  }
+
+  # Install a newer version of Node.js for the Slack bridge and web IRC app
+  apt::key { 'nodejs':
+    id     => '9FD3B784BC1C6FC31A8A0A1C1655A0AB68576280',
+    source => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key';
+  }
```

Comment:
I don't think backporting for all servers would be a good idea, especially given the [number of API changes](https://github.com/nodejs/node/wiki/API-changes-between-v0.10-and-v4) between 0.10 and 4.0. We don't want to break any node apps we have running on apphosting just for this. I like the sounds of the first option the best, but I'm not really sure what it would involve.